### PR TITLE
DOC-1251 Offloading docs improvements

### PR DESCRIPTION
**Pull request by claireimprobable**

### CODE REVIEW

File: SpatialGDK/Documentation/content/tutorials/offloading-tutorial/offloading-gameplay-changes.md [hunk 1]
```diff
@@ -49,13 +49,11 @@ Another thing that needs to be taken into consideration when implementing gamepl
...
-In the example project,  the Player characters and the CrashBots have the same base class, we had to create a child blueprint called BP_CrashBot to ensure that the offloading configuration setup correctly.
+In the example project, the `Player` characters and the `CrashBots` have the same base class, we had to create a child blueprint called `BP_CrashBot` to ensure that the offloading configuration setup correctly.
 
-In addition to the CrashBot blueprint, the example project had to create an additional version of the CrashBot named  BP_CrashBot_PlacedInWorld. This was to work around the issue of creating controllers first when the authority was gained.
+In addition to the `CrashBot` blueprint, the Example Project had to create an additional version of the CrashBot named `BP_CrashBot_PlacedInWorld`. This was to work around the issue of creating the controllers first when the authority was gained.
 
-Mention weapon to allow it to be used for the bots.
-
-### Ensure that code get executed on the correct server worker
+### Ensure that your code get executed on the correct server worker
```
Comment (eddyrainy) [inline SpatialGDK/Documentation/content/tutorials/offloading-tutorial/offloading-gameplay-changes.md]: Ensure that your code gets executed on the correct server worker

File: SpatialGDK/Documentation/content/tutorials/offloading-tutorial/offloading-intro.md [hunk 2]
```diff
@@ -2,15 +2,15 @@
 
 # Tutorials and guides
 
-## Offloading example project
+## Offloading Example Project
+> **This tutorial uses the Example Project from the GDK's [setup guide]({{urlRoot}}/content/get-started/example-project/exampleproject-intro).**</br>
 
-The UnrealGDK Example project contains a map called `CrashBot_Gym` that demonstrates how to set up a gameplay feature using offload to separate workers.
+The UnrealGDK Example project contains a map called `CrashBot_Gym` that demonstrates how to offload gameplay features to separate workers.
```
Comment (eddyrainy) [inline SpatialGDK/Documentation/content/tutorials/offloading-tutorial/offloading-intro.md]: Hey, I used **set up a feature using offloading** to be consistent with one of our topic titles.

File: SpatialGDK/Documentation/content/tutorials/offloading-tutorial/offloading-intro.md [hunk 3]
```diff
@@ -2,15 +2,15 @@
...
 
-The scenario in the `CrashBot_Gym` contains a set of turrets that attempt to attack any nearby players or AI-controlled bots. The turrets are offloaded to a server-worker called `AIWorker` and the bots are offloaded to a server-worker called `CrashBotWorker`.
+The scenario in the `CrashBot_Gym` contains a set of turrets that will attempt to attack any nearby players or AI-controlled bots. The turrets are offloaded to a worker called `AIWorker` and the bots are offloaded to a worker called `CrashBot_Worker`. This section will go through the details of what was required to enable offloading in this scenario.
```
Comment (eddyrainy) [inline SpatialGDK/Documentation/content/tutorials/offloading-tutorial/offloading-intro.md]: I don't think we need this: This section will go through the details of what was required to enable offloading in this scenario.

We already have **In this tutorial, you will go through the steps to enable offloading.**

File: SpatialGDK/Documentation/content/tutorials/offloading-tutorial/offloading-intro.md [hunk 4]
```diff
@@ -21,5 +21,4 @@ In this tutorial, you will go through the steps to enable offloading.
 _2019-07-30 Page added as limited editorial review_
 <br/>
 <br/>
-[//]: # (TODO: https://improbableio.atlassian.net/browse/DOC-1142)
-------------
\ No newline at end of file
+------------
```
Comment (eddyrainy) [inline SpatialGDK/Documentation/content/tutorials/offloading-tutorial/offloading-intro.md]: We need to keep this: [//]: # (TODO: https://improbableio.atlassian.net/browse/DOC-1142)

This is a reminder to tech writers, but not visible to users.

File: SpatialGDK/Documentation/content/workers/offloading-concept.md [hunk 5]
```diff
@@ -2,11 +2,11 @@
...
 
 In Unreal’s native single-server architecture, the server runs many of the major game systems such as physics simulation, AI decision-making, and navigation. It also processes and validates input from game clients. Using the GDK allows you to execute latency-tolerant systems on a separate server-worker instance.
 
-You can think of _offloading_ as splitting up computation-heavy systems to run on separate server-worker instances. As a result, some CPU-intensive game systems that cannot easily use multithreading with the Unreal’s native architecture are no longer limited by the processing power of a single server. In addition, not all input from game clients needs to be processed by the same server-worker instance.
+You can think of _offloading_ as splitting up computationally heavy systems to run on separate server-worker instances. As a result, some CPU-intensive game systems that cannot easily use multithreading with the Unreal’s native architecture are no longer limited by the processing power of a single server. In addition, not all input from game clients needs to be processed by the same server-worker instance.
```
Comment (eddyrainy) [inline SpatialGDK/Documentation/content/workers/offloading-concept.md]: I think **computation-heavy systems** is okay.

File: SpatialGDK/Documentation/content/workers/set-up-offloading.md [hunk 6]
```diff
@@ -2,18 +2,19 @@
...
+To set up your game feature for offloading, in addition to configuring your Unreal game server, you also need to start an Unreal process for your offloaded worker types, both locally in PIE and in the cloud. In particular, you specify the number of offloaded servers to start and their attributes in the Unreal Editor, and configure Actor groups to set the authority configuration for the Actors.
 
 Before you start, review the following offloading workflows. You can check each step for detailed information:
 
 1. Create a [worker configuration file]({{urlRoot}}/content/glossary#worker-configuration-file): You create a worker configuration file for the new offloaded worker type.
+    **Notes**:
+    - If you launch a local deployment through PIE within the editor, the Unreal GDK adds the worker configuration file for your new offloaded worker type, so you can skip this step.
+    - If you launch a local deployment through PIE within the editor, the Unreal GDK adds the worker configuration file for your new offloaded worker type, so you can skip this step.
+2. Specify the launch configuration:
```
Comment (eddyrainy) [inline SpatialGDK/Documentation/content/workers/set-up-offloading.md]: Is this duplicated?

    - If you launch a local deployment through PIE within the editor, the Unreal GDK adds the worker configuration file for your new offloaded worker type, so you can skip this step.
    - If you launch a local deployment through PIE within the editor, the Unreal GDK adds the worker configuration file for your new offloaded worker type, so you can skip this step.

File: SpatialGDK/Documentation/content/tutorials/offloading-tutorial/offloading-gameplay-changes.md [hunk 7]
```diff
@@ -49,13 +49,11 @@ Another thing that needs to be taken into consideration when implementing gamepl
 
 The solution to this is to create a new subclass of the weapon that can be assigned to the actor group that the offloaded AI belongs to.
 
-In the example project,  the Player characters and the CrashBots have the same base class, we had to create a child blueprint called BP_CrashBot to ensure that the offloading configuration setup correctly.
+In the example project, the `Player` characters and the `CrashBots` have the same base class, we had to create a child blueprint called `BP_CrashBot` to ensure that the offloading configuration setup correctly.
```
Comment (padhansell) [inline SpatialGDK/Documentation/content/tutorials/offloading-tutorial/offloading-gameplay-changes.md]: ```In the example project, the `Player` characters and the `CrashBots` have the same base class, we had to create a child blueprint called `BP_CrashBot` to ensure that the offloading configuration setup correctly.```

This sentence don't make no sense

File: SpatialGDK/Documentation/content/workers/offloading-concept.md [hunk 8]
```diff
@@ -47,8 +47,9 @@ Before you offload Actors, consider the following scenarios that you need to upd
     However, this might cause issues when the initialization logic in the calls such as `BeginPlay` and `PostInitializeComponent` is executed on the wrong server-worker instance. You can usually work around these issues using callbacks / RepNotify-s on the main Unreal server-worker instance to defer the execution of such logic until when the correct server-worker instance has authority over the offloaded Actor.
 
 <br/>------------<br/>
+_2019-08-12 Page updated with limited editorial review_
```
Comment (ElleEss) [inline SpatialGDK/Documentation/content/workers/offloading-concept.md]: @claireimprobable - added that page updated.

File: SpatialGDK/Documentation/content/tutorials/offloading-tutorial/offloading-intro.md [hunk 9]
```diff
@@ -19,7 +17,8 @@ In this tutorial, you will go through the steps to enable offloading.
 
 <br/>------------<br/>
 _2019-07-30 Page added as limited editorial review_
+_2019-08-12 Page updated with limited editorial review_
```
Comment (ElleEss) [inline SpatialGDK/Documentation/content/tutorials/offloading-tutorial/offloading-intro.md]: @claireimprobable FYI

File: SpatialGDK/Documentation/content/tutorials/offloading-tutorial/offloading-gameplay-changes.md [hunk 10]
```diff
@@ -77,13 +75,15 @@ void UEquippedComponent::EndPlay(const EEndPlayReason::Type EndPlayReason)
...
 ### Conclusion
 
-By using offloading, you can scale your CPU heavy tasks by splitting them to run on instances of different server-workers. You don’t need to worry about the challenges involved with migrating entities across worker boundaries.
+By using offloading, you can scale your CPU heavy tasks by splitting them to run on instances of different server-workers. You don’t need to worry about the challenges involved with migrating entities across worker boundaries. 
+
 
 Meanwhile, offloading introduces a set of constraints that you need to take care of.  When you set up your feature using offloading properly, you can have significant scaling opportunities.
 
 <br/>------------<br/>
+_2019-08-12 Page updated with limited editorial review_
```
Comment (ElleEss) [inline SpatialGDK/Documentation/content/tutorials/offloading-tutorial/offloading-gameplay-changes.md]: @claireimprobable  - FYI - Added the update status to the page changelogs where there are changes more than the odd word.
Thanks for updating. Will give it a proper look when I do the JIRA ticket. (Next task for me.)

**Comment by ernopp** [comment]
this is made obsolete by #1325

**Pull request closed without merging**